### ISSUE TITLE
Killing Zez-Kai Doesn't Give DS Points

### ISSUE THREAD

**Issue by Mako88** [KOTORCommunityPatches/TSL_Community_Patch#9]
Reported by Sniggles on Discord, apparently killing the Jedi Master Zez-Kai doesn't give you DS points.

**Comment by JCarter426** [comment]
Yeah, that seems to be the case, and it's more than that. His whole dialogue tree is... wrong.

He always asks if you're going to kill him, regardless of your alignment or other choices. There are a couple non-sequitors in the conversation due to nodes that probably shouldn't have been linked together. I found some infinite loops as well - you can tell him you have no more questions, then ask more questions, and go back and forth forever.

All this can be fixed, but a proper fix might involve a hard DLG replacement. I'm not sure what the compatibility situation would be with that.

Revan's gender is wrong on one line, too, but that's a TLK problem. TSLRCM might already fix that, not sure.

**Comment by DarthParametric** [comment]
This should be an unmolested TSLRCM version of the DLG I believe.

[TSLRCM_zezkaiel.zip](https://github.com/KOTORCommunityPatches/TSL_Community_Patch/files/2979129/TSLRCM_zezkaiel.zip)

**Comment by JCarter426** [comment]
Looks like all the same issues in the DLG are there. So the question is if TSLPatcher can't patch all the fixes (I suspect not) then would a hard replacement cause problems.

**Comment by DarthParametric** [comment]
Shouldn't do. Should just be a straight module injection with a hard overwrite. Presumably we are mandating that TSLRCM is a perquisite, so the module will already be there anyway.

**Comment by JCarter426** [comment]
That's what I mean, it would entirely replace the TSLRCM version. I don't know if that would be an issue if other mods then tried to patch it based on TSLRCM's copy instead of the community patch's.

**Comment by DarthParametric** [comment]
It would be an issue in that instance no doubt, but I think the gains outweigh the edge case of someone trying to mod it. Besides, if there is such a mod, dollars to donuts it dumps a copy in the Override anyway.
